core/src/test/scala/unit/kafka/server/AbstractApiVersionsRequestTest.scala

```diff
@@ -74,7 +74,6 @@ abstract class AbstractApiVersionsRequestTest(cluster: ClusterInstance) {
       assertEquals(MetadataVersion.latestTesting().featureLevel(), apiVersionsResponse.data().supportedFeatures().find(MetadataVersion.FEATURE_NAME).maxVersion())
 
       assertEquals(0, apiVersionsResponse.data().supportedFeatures().find(GroupVersion.FEATURE_NAME).minVersion())
```

Comment:
this should be removed too. Also, the expected value line#72 should be changed to `1` as `group.version` is reverted